netjsonconfig/backends/openwrt/schema.py

```diff
@@ -386,10 +419,579 @@
+                                    "default": False,
+                                    "format": "checkbox",
+                                    "propertyOrder": 4,
+                                },
+                                "input": {
+                                    "allOf": [
+                                        {"$ref": "#/definitions/zone_policy"},
+                                        {
+                                            "title": "input",
+                                            "description": "default policy for incoming zone traffic",
+                                            "propertyOrder": 5,
+                                        },
+                                    ]
+                                },
+                                "output": {
+                                    "allOf": [
+                                        {"$ref": "#/definitions/zone_policy"},
+                                        {
+                                            "title": "output",
+                                            "description": "default policy for outgoing zone traffic",
+                                            "propertyOrder": 6,
+                                        },
+                                    ]
+                                },
+                                "forward": {
+                                    "allOf": [
+                                        {"$ref": "#/definitions/zone_policy"},
+                                        {
+                                            "title": "forward",
+                                            "description": "default policy for forwarded zone traffic",
+                                            "propertyOrder": 7,
+                                        },
+                                    ]
+                                },
+                            },
+                        },
+                    },
+                    "rules": {
+                        "type": "array",
+                        "title": "Rules",
+                        "propertyOrder": 7,
+                        "items": {
+                            "type": "object",
+                            "title": "Rules",
+                            "additionalProperties": True,
+                            "required": ["src", "target"],
+                            "properties": {
+                                "name": {
+                                    "type": "string",
+                                    "title": "name",
+                                    "description": "name of the rule",
+                                    "propertyOrder": 1,
+                                },
+                                "src": {
+                                    "type": "string",
+                                    "title": "src",
```

Comment:
let's try to put human readable labels in each `title`, eg: `source` or `source zone`, this suggestion applies to all other occurrences of `title`